README.md

```diff
@@ -4,50 +4,74 @@
 CLI tool to scan your postgres database, and output DBML.
 - [pg-to-dbml](#pg-to-dbml)
     - [Description](#description)
-  - [Overview](#overview)
-    - [Resources](#resources)
-    - [Download](#download)
+    - [Tech Stack](#tech-stack)
+    - [Download and install](#download-and-install)
     - [How to use the cli](#how-to-use-the-cli)
     - [Example Usage](#example-usage)
   - [Linting](#linting)
+  - [Debugging](#debugging)
   - [Changelog](#changelog)
 
-## Overview
-
-The purpose of the reference application is to provide a starting point for Node.js development that encapsulates good tools and best practices such as TypeScript, configuration management, linting, testing, and documentation.
-
-### Resources
+### Tech Stack
 
 - [NodeJS](https://nodejs.org/en/docs/)
 - [Yargs](http://yargs.js.org/)
 - [PG](https://node-postgres.com/)
 
-### Download
-`npm i 
+### Download and install
+
+`npm imstall pg-to-dbml` 
 
 ### How to use the cli
 
+To see all the commands and options, run `pg-to-dbml --help`
+
 ```
-Usage: pg-to-dbml [options]
+Usage: pg-to-dbml [command] [options]
+
+Commands:
+  to-dbml  default command. connects to pg db directly and creates dbml
```

Comment:
Is `to-dbml` a subcommand here, or just a typo maybe?

Reply:
it is the name of the command, though you don't have to be explicit with it because it is the default command. I did this to make it possible to add additional commands.

Reply:
Ok nice, yeah having a default subcommand sounds like the way to go.